app.coffee

```diff
@@ -66,6 +69,8 @@ app.post "/webhook", (req, res) ->
 
   if slack.userInfoById(req.body.user_id)
     avatar = slack.userInfoById(req.body.user_id)['profile']['image_72']
+    if process.env.ADMIN_NICK == "full"
+      adminNick = slack.userInfoById(req.body.user_id)['profile']['real_name']
```

Comment:
@caske33 You will need to put up an else case here when `process.env.ADMIN_NICK` is not defined. Otherwise `nick` in request below will get undefined.